src/CacheException.php

```diff
@@ -5,6 +5,6 @@
 /**
  * Exception interface for all exceptions thrown by an Implementing Library.
  */
-interface CacheException
+interface CacheException extends \Throwable
```

Comment:
Looks like the same change was missed for the InvalidArgumentException?

Reply:
The InvalidArgumentException implements CacheException, right?

Reply:
Sorry, nvm!

Reply:
I don't know how to close this one, approving doesn't help apparently.